src/components/ChannelInner.js

```diff
@@ -199,8 +217,10 @@ export class ChannelInner extends PureComponent {
 
   loadMoreThread = async () => {
     // prevent duplicate loading events...
-    if (this.state.threadLoadingMore) return;
```

Comment:
Using debouncer instead.

Reply:
`FlatList` makes unnecessary calls to load more function because of which `loadMoreThread ` won't be accessible to us when its actually needed (in case of connection recovery or when there are changes to thread object). And thus removing this condition and replacing it with debouncer!!